.github/pull.yml

```diff
@@ -0,0 +1,14 @@
+version: "1"
+rules:
+  - base: master
+    upstream: UncleGoogle:master
+    mergeMethod: none  # Optional, one of [none, merge, squash, rebase, hardreset], Default: none.
+    mergeUnstable: false    # Optional, merge pull request even when the mergeable_state is not clean. Default: false
+    # assignees:              # Optional
+    reviewers:              # Optional
+      - urwrstkn8mare
+      - AndrewDWhite
+    conflictReviewers:      # Optional, on merge conflict assign a reviewer
+      - UncleGoogle
```

Comment:
I don't expect any conflicts... and I shouldn't be a reviewer for my own repo at all probably :D But I'm not sure what this conflictReview is -- din't found any info if this is normal reviewer with some extra rules or what. I've put it here to check and because I would probably be best person to resolve conflicts with my own repo....

Reply:
I think this just adds the ability to assign a reviewer not normally assigned to the repo in case a conflict is found.

>  I would probably be best person to resolve conflicts with my own repo

Yep